overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/ignition-ostree-drop-luks.service

```
@@ -0,0 +1,20 @@
+[Unit]
+Description=Ignition OSTree: Drop SE LUKS keys from ramdisk
```

Maybe

```suggestion
Description=Ignition OSTree: Delete Secure Execution LUKS Keys
```

for consistency.